tools/Datadog.Core.Tools/Datadog.Core.Tools.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;net461;netcoreapp2.1;netcoreapp3.0</TargetFrameworks>
```

Comment:
nit: These can probably be reduced to something like `net45;netstandard2.0` but no need to block on this.